Library/Homebrew/system_config.rb

```diff
@@ -136,6 +149,7 @@ def dump_verbose_config(f = $stdout)
       f.puts "Perl: #{describe_perl}"
       f.puts "Python: #{describe_python}"
       f.puts "Ruby: #{describe_ruby}"
+      f.puts "System Ruby: #{describe_system_ruby}"
```

Comment:
Also, I think it may be a good idea to rename `Git` to `Homebrew Git` as they are in the same category, i.e., tools used by Homebrew itself.